src/bramble/client/main.js

```diff
@@ -122,14 +122,26 @@ define([
 
     // Expose Filer for Path, Buffer, providers, etc.
     Bramble.Filer = Filer;
-    var _fs = new Filer.FileSystem();
+
+    var _fs;
+    var useTemporaryStorage = location.search === "?BrambleMemoryFileSystem";
```

Comment:
I think right now a bunch of other query string parameters are also passed in, so this won't work. You'll need to use a regex to check if the querystring contains the right search parameter. Also, as @humphd mentioned, rename the query parameter to `temporaryStorage`.

Try this instead:
```js
var useTemporaryStorage = /(temporary[sS]torage[=&])|(temporary[Ss]torage$)/.test(window.location.search);
```

Reply:
Thanks for the snippet. I'll change and commit the code right now. One question: this regex checks if the query is either 'temporaryStorage' or 'temporarystorage'?

Reply:
yep 👍